### ISSUE TITLE
Add a `manual` detection mode

### ISSUE THREAD

**Issue by sellout** [LionyxML/auto-dark-emacs#73]
I want a way for Auto-Dark to manage the pairs of themes & hooks, but to not do any auto-detection of system mode changes.

This is useful for testing, but also for niche systems that don’t have any of the existing systems. This is already _possible_, but some changes to make it more ergonomic:
- an `auto-dark-toggle-appearance` command, so no internal functions need to be called; and
- a way to get `auto-dark-mode` to not try to re-determine the detection method (either adding a `manual` detection method, or moving the `auto-dark--determine-detection-method` call to the `defcustom` form and treating `nil` as the manual mode)[^1].

[^1]: I prefer using `nil` for manual mode (read as “there is no detection method”), but perhaps it’s desirable to have a case where re-enabling the theme tries to re-detect the method (as it does now). I think I would prefer having a public `auto-dark-determine-detection-method` command that users can explicitly call if they want to re-detect.

**Comment by sellout** [comment]
Ah, and manual mode should also not run the timer.

**Comment by LionyxML** [comment]
Thanks again for all this work @sellout ! Great great stuff here!

I think the pipeline passed:  https://garnix.io/commit/988121d61e1770cac4105b04218c19fa4c7126b8

Is there a way to make garnix integrated with github? So everybody can wee the status? Like a badge or some integration with github actions?

**Comment by sellout** [comment]
> Is there a way to make garnix integrated with github? So everybody can see the status? Like a badge or some integration with github actions?

Yeah, both of those things – you’ll need to authorize the garnix app (https://github.com/apps/garnix-ci), which then gives you GitHub checks for garnix CI and `[![built with garnix](https://img.shields.io/endpoint.svg?url=https%3A%2F%2Fgarnix.io%2Fapi%2Fbadges%2FLionyxML%2Fauto-dark-emacs)](https://garnix.io/repo/LionyxML/auto-dark-emacs)` will give you a badge (with useful information once the app is authorized).

**Comment by LionyxML** [comment]
Noice! 

Just added them to the development branch.
